src/components/Dashboard/Dashboard.jsx

```diff
@@ -6,7 +6,7 @@ import Message from '../Message/Message'
 import PlayerCards from '../Cards/PlayerCards';
 import style from './Dashboard.module.css';
 import axios from 'axios';
-import { setGameState } from '../../store/actions';
+import { setGameState, setMessage } from '../../store/actions';
```

Comment:
I know this isn't from this commit but you don't usally keep actions in the store folder. You end up with some nasty imports and it means the actions aren't close to the files they relate to.